### ISSUE TITLE
Ability to repopulate cache from command line

### ISSUE THREAD

**Issue by fizyk** [stevearc/pypicloud#143]
We've had an issue where we've migrated pypicloud isntance to new type with redis locally, and we've lost cache data. ow during start pypicloud tries to repopulate it, however it was too long

Wer'e using a systemd to start pypicloud and it determined the pypicloud as timed out. It would be great to be able to re-populate cache from command line. I actually bypassed this by starting pypicloud by hand and after it finally refreshed the cache, started it automatically again.

**Comment by stevearc** [comment]
Would exposing a specific command to repopulate the cache be better than simply running the server from the command line? They seem about the same to me.

**Comment by fizyk** [comment]
Yes, since now we've got a ton of packages versions already (uploading a stable and pre-releases every ci build). 
And apparently systemd's unit through which we're starting pypicloud have a timout (30 seconds or so). Apparently after the start times out, the command isn't simply left running, instead is brutally killed.

Trying to restart only runs the cache rebuild from the begging which fails again.
https://www.freedesktop.org/software/systemd/man/systemd.service.html
TimeoutStartSec could be set to infinity, however that would't make the service start rather underterministic.
Other solutions apart from a command would be to get the cache rebuild done in a thread, and let the pypicloud start along the cache being rebuilt.

**Comment by stevearc** [comment]
I like the idea of spawning a thread to do the rebuild. It's an option now that the caches have the ability to do a live-reload. I'll look into that for the next release

**Comment by dmarkey** [comment]
Hey, is there any way to do this via the docker image currently?
